docker-compose.yml

```diff
@@ -10,31 +11,44 @@ services:
     #     acapy_url: https://github.com/Indicio-tech/aries-cloudagent-python@c1fed3c13d33e65979b08dd1eaf79dc84e3ce504
     ports:
       - "3001:3001"
+    volumes:
+      - ./tunnel_endpoint.sh:/tunnel_endpoint.sh:ro,z
+    environment:
+        AGENT_TUNNEL_ENDPOINT: http://tunnel-alice:4040
+        TAILS_TUNNEL_ENDPOINT: http://tunnel-tails:4040
+    entrypoint: >
+      /bin/sh -c '/tunnel_endpoint.sh aca-py "$$@"' --
     command: >
       start -it http 0.0.0.0 3000
         --label Alice
         -ot http
         -e http://alice:3000
+        --tails-server-base-url http://tails:6543
```

Comment:
This should be removed since it's being set by the `tunnel_endpoint.sh` script as an environment variable.